test/DynamoCoreWpfTests/ViewExtensions/PythonMigrationViewExtensionTests.cs

```diff
@@ -0,0 +1,235 @@
+            Assert.AreNotEqual(ironPythonWorkspaceId, this.ViewModel.CurrentSpace.Guid);
+            
+
+            Open(examplePathIronPython);
+            Assert.AreEqual(ironPythonWorkspaceId, this.ViewModel.CurrentSpace.Guid);
+            var secondGraphIronPythonDialog = GetOpenWindowsOfType<IronPythonInfoDialog>(this.View);
+
+            // Assert
+            Assert.AreEqual(0, secondGraphIronPythonDialog.Count());
+        }
+
+        /// <summary>
+        /// This tests checks if the extension can detect IronPython nodes in the graph 
+        /// </summary>
+        [Test]
+        public void CanDetectIronPythonNodesInGraph()
+        {
+            // Arrange
+            RaiseLoadedEvent(this.View);
+            var extensionManager = View.viewExtensionManager;
+            extensionManager.Add(viewExtension);
+
+            // Act
+            // open file
+            Open(@"core\python\python.dyn");
+
+            var pythonMigration = extensionManager.ViewExtensions.Where(x => x.Name == viewExtension.Name).Select(x=>x).First() as PythonMigrationViewExtension;
+
+            // Assert
+            Assert.IsTrue(pythonMigration.PythonDependencies.ContainsIronPythonDependencies());
+        }
+
+        /// <summary>
+        /// Checks if pressing the `More Information` button on the IronPythonInfoDialog window
+        /// will open the DocumentaionBrowser viewExtension.
+        /// </summary>
+        [Test]
+        public void CanOpenDocumentationBrowserWhenMoreInformationIsClicked()
+        {
+            // Arrange
+            RaiseLoadedEvent(this.View);
+            var extensionManager = View.viewExtensionManager;
+            extensionManager.Add(viewExtension);
+
+            // Act
+            // open file
+            var examplePath = Path.Combine(UnitTestBase.TestDirectory, @"core\python", "python.dyn");
+            Open(examplePath);
+
+            var ironPythonDialog = GetOpenWindowsOfType<IronPythonInfoDialog>(this.View).First();
+            var viewExtensionTabsBeforeBtnClick = this.View.ExtensionTabItems.Count;
+
+            ironPythonDialog.MoreInformationBtn.RaiseEvent(new RoutedEventArgs(Button.ClickEvent));
+
+            // Assert
+            Assert.AreNotEqual(viewExtensionTabsBeforeBtnClick, this.View.ExtensionTabItems.Count);
+            Assert.IsTrue(this.View.ExtensionTabItems
+                .Any(x => x.Header.ToString() == "Documentation Browser"));
+        }
+
+        #region Helpers
+        public static void RaiseLoadedEvent(FrameworkElement element)
+        {
+            MethodInfo eventMethod = typeof(FrameworkElement).GetMethod("OnLoaded",
+                BindingFlags.Instance | BindingFlags.NonPublic);
+
+            RoutedEventArgs args = new RoutedEventArgs(FrameworkElement.LoadedEvent);
+
+            eventMethod.Invoke(element, new object[] { args });
+        }
+
+        public static IEnumerable<T> GetOpenWindowsOfType<T>(Window view)  where T : Window
```

Comment:
this is repeated in the other test too, suggest moving it to an extension class that's `internal` to this test project + rename as suggested in other comment